src/js/serial_backend.js

```diff
@@ -237,6 +238,10 @@ function finishClose(finishedCallback) {
     // unlock port select & baud
     $('div#port-picker #port').prop('disabled', false);
     if (!GUI.auto_connect) $('div#port-picker #baud').prop('disabled', false);
+    const selectedPort = $('div#port-picker #port option:selected');
+    if (selectedPort?.data()?.isManual) {
+        $('#port-override-option').show();
+    }
```

Comment:
Should we call `GUI.updateManualPortVisibility()` instead?

Reply:
Definitely makes sense - thank you for the suggestion! I also took the liberty to update the `GUI.updateManualPortVisibility` logic to only show the `#port-override-option` (manual port input) if a connection has not been already established (based on `isConnected`).